### PR TITLE
[FIX] move bep 13 to "other BEP" table

### LINKED ISSUE #296: move BEP13 out of main BEP tables as it was merged into BEP12

### PR DESCRIPTION
**Pull request by Remi-Gau**
closes #296

### CODE REVIEW

File: _pages/get_involved.md [hunk 1]
```diff
@@ -70,9 +70,10 @@ can be found in the BIDS [specification](https://bids-specification.readthedocs.
...
-| --------------------------------------------------------------- | ------------ | ---------------------------------------------- | ------------------------------------------------------------------------- |
-| [BEP015](https://bids.neuroimaging.io/bep015){:target:"_blank"} | Mapping file | Eric Earl, Camille Maumet, and Vasudev Raguram | [File mapper](https://github.com/DCAN-Labs/file-mapper){:target:"_blank"} |
+| Extension label                                                 | Title                          | Moderators/leads                               | Outcome                                                                   |
+| --------------------------------------------------------------- | ------------------------------ | ---------------------------------------------- | ------------------------------------------------------------------------- |
+| [BEP012](https://bids.neuroimaging.io/bep012){:target:"_blank"} | Resting state fMRI derivatives | Steven Giavasis                                | Merged into BEP012                                                        |
```
Comment (sappelhoff) [inline _pages/get_involved.md:78]: shouldn't it say BEP013 here?

Comment (Remi-Gau) [reply _pages/get_involved.md:78]: oh god... I have the attention span of an oyster!!!

Comment (Remi-Gau) [inline _pages/get_involved.md:78]:
```suggestion
| [BEP013](https://bids.neuroimaging.io/bep013){:target:"_blank"} | Resting state fMRI derivatives | Steven Giavasis                                | Merged into BEP012                                                        |
```

Comment (Remi-Gau) [reply _pages/get_involved.md]: should be fixed
thanks for catching this

**Review by sappelhoff — APPROVED** [review]

**Review by effigies — APPROVED** [review]

**Pull request merged**